romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Ok, maybe the parsing library does this. And - yes - technically a YAML parser can parse JSON. But as a human, it's weird to see a JSON formatted file with a .yaml extension. Isn't that surprising? It was surprising to me.

If we call the file *.yaml then why don't we format it in plain YAML?